src/main/scala/nl.knaw.dans.easy.bagstore/server/BagsServletComponent.scala

```diff
@@ -30,7 +30,7 @@ trait BagsServletComponent extends DebugEnhancedLogging {
 
   val bagsServlet: BagsServlet
 
-  trait BagsServlet extends ScalatraServlet with ServletUtils {
+  trait BagsServlet extends ScalatraServlet with ServletUtils with ServletEnhancedLogging {
```

Comment:
There are multiple servlets in this microservice. Why only add the logging to this servlet?